DTDL/edge/modbus/modbus-source.json

```diff
@@ -0,0 +1,75 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:io:tributech:source:modbus;2",
```

Comment:
`dtmi:io:tributech:source:modbus;1` exists in old(v1) config. When I try to add this config on Demeter on dev-x i get 
`dtmi already exists` error.

Should I use `2` or should I propose a different id?
@Badabunga @LukasSchwaiger

Reply:
please change the version number back to 1 dtmi:io:tributech:source:modbus:1